src/routes/index.ts

```diff
@@ -8,7 +8,7 @@ import categoryRoutes from './products.category.routes';
 import productRoutes from './products.routes';
 import vendorRoutes from './vendor.routes';
 import wishListRoute from './whishlist.route';
-
+import adminRoute from './admin.disable.account.route';
```

Comment:
Rename this file to just admin.routes.ts We'll be using it to add other admin access only routes
```suggestion
import adminRoute from './admin.routes';
```